eclipse-mosquitto.yaml

```diff
@@ -0,0 +1,110 @@
+package:
+  name: eclipse-mosquitto
+  version: 2.0.18
+  epoch: 0
+  description: Eclipse Mosquitto - An open source MQTT broker
+  copyright:
+    - license: EPL-2.0 AND BSD-3-Clause
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+      - cjson
+      - libwebsockets-dev
+      - mosquitto
+      - mosquitto-clients
+      - mosquitto-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: ^(\d+\.\d+)\.\d+$
+    replace: "$1"
+    to: major-minor-version
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 3923526c6b4c048bbecad2506c4c9963bc46cd36
+      repository: https://github.com/eclipse/mosquitto
+      tag: v${{package.version}}
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}
+      mkdir -p ${{targets.contextdir}}/mosquitto/config
+      mkdir -p ${{targets.contextdir}}/mosquitto/data
+      mkdir -p ${{targets.contextdir}}/mosquitto/log
+      install -Dm644 docker/${{vars.major-minor-version}}-openssl/docker-entrypoint.sh ${{targets.contextdir}}/docker-entrypoint.sh
+      install -Dm644 docker/${{vars.major-minor-version}}-openssl/mosquitto-no-auth.conf ${{targets.contextdir}}/mosquitto-no-auth.conf
```

Comment:
Does this need to build anything?  Looks like this is just copying config?  I noticed upstream has a binary that gets built? https://github.com/eclipse/mosquitto/blob/15292b20b0894ec7c5c3d47e4b22ee9d89f91132/Makefile#L59

Reply:
Or is it the same as https://github.com/wolfi-dev/os/blob/9a2d70dd0bb34f56ba35329da2a6bb3f9e9ebbbc/mosquitto.yaml#L28?  If it is maybe we should switch that to git-checkout and add a subpackage for eclipse-mosquitto?

Reply:
This is something I thought about at the beginning, then I wasn't sure if this was using the same version as the mosquitto package, so I decided to go with another one but yes this might work.

Reply:
Any thoughts @jamonation?